index.html

```diff
@@ -206,5 +206,81 @@ <h4>Skills</h4>
         
     </section>
 </section>
+
+<footer>
+        <div class="footspace">
+           <!-- <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+           <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+           --></div> 
+        <div class=" box name"><h2>Get Started</h2></div>
```

Comment:
- [ ] Our Font size, family, and weight are a bit off, I made a video demonstrating how to get the exact measurement from Figma, please check it [here](https://www.loom.com/share/ab7e9fcd9e014b74a0f1d466c9960f99)

Reply:
When I use the same font family and some of the same values, I get a linter error. 
So I need to change them.